coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "line": 82.81
+    "line": 82.78
```

Comment:
This is why `refuse_coverage_drop` is bad in development. This drop happened who knows how long ago, but it wasn't written to the file because we told SompleCov not to.

Reply:
SOMPLECOV